plugins/hydra_ray_launcher/hydra_plugins/hydra_ray_launcher/_core_aws.py

```diff
@@ -147,7 +151,7 @@ def launch_jobs(
 
             ray_exec(
                 launcher.ray_yaml_path,
-                launcher.docker_enabled,
+                launcher.ray_cfg.run_env,
```

Comment:
no type error here? ray_cfg.run_env is the Enum, but the function wants a strings.